packages/gestalt/src/Card.test.js

```diff
@@ -3,20 +3,25 @@ import React from 'react';
 import renderer from 'react-test-renderer';
 import Card from './Card.js';
 
-const snapshot = component => {
-  const tree = renderer.create(component).toJSON();
```

Comment:
https://github.com/jest-community/eslint-plugin-jest/blob/master/docs/rules/valid-expect.md didn't work well when we extracted this into its own function